pyproject.toml

```diff
@@ -6,14 +6,15 @@ authors = ["proyecto-nutria <contact@proyectonutria.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-"discord.py" = "^2.0.1"
+"discord.py" = "2.4.0"
```

Comment:
Should we stick to a fixed version of `discord.py`?

Reply:
Not sure, with the lock file should not be a problem, but when testing I noticed that this new version broke our tests. So my take will be let this as fixed version as is the "main  library", while we can let the others as now.